client/src/pages/Dashboard/components/SurveyCard.module.css

```diff
@@ -0,0 +1,16 @@
+.actions {
+  justify-content: space-between;
+}
+
+.active {
+  background: rgb(169, 235, 169) !important;
```

Comment:
I was under the impression that if you're writing `!important` then you need to restructure your CSS ;)